ACD-416 (#6)
Actindo-AG/Documentation: RetailSuiteAccounting/Operation/15_ProvideAccountingData.md

[!!User Interface export DATEV CSV](../UserInterface/03b_Export.md#datev-csv)  
[!!User Interface GDPdU export](../UserInterface/03b_Export.md#gdpdu-export)  
[!!User Interface CVS format definition](../UserInterface/03c_CSVFormatDef.md)  
[!!User Interface BWA](../UserInterface/01_Book.md#bwa)  
[!!User Interface BWA assignment](../UserInterface/02i_BWAAssignment.md)  
[!!Manage the BWA reports](../Integration/07_ManageBWAReports.md)  

# Provide the accounting data

The *Accounting* module offers an export function to export data in different formats. The most widely used financial formats for accounting software are supported.

> Exporting data may be necessary to perform an external audit, create a balance sheet or prepare the annual accounts. These tasks are usually performed at the end of a fiscal year. The system also includes a reporting functionality to create different report types and export them in PDF format.

Reviewer: ```suggestion
Exporting data may be necessary to perform an external audit, create a balance sheet or prepare the annual accounts. These tasks are usually performed at the end of a fiscal year. The system also includes a reporting functionality to create different report types and export them as a PDF.
```

## Export the data for DATEV

Accounting data can be exported into a DATEV CSV compatible file to be later imported in DATEV software for further processing.  

#### Prerequisites

 - The accounting is configured via the configuration wizard, see [Run the accounting wizard](../Integration/01_RunAccountingWizard.md).
 - A fiscal year is selected, see [Select a fiscal year](./01_SelectFiscalYear.md).

#### Procedure

*Accounting > Select fiscal year > Extras > Tab EXPORT*

![Export](../../Assets/Screenshots/RetailSuiteAccounting/Extras/Export/Export.png "[Export]")

1. Click the [DATEV CSV] button.    
The *Export: DATEV CSV: Step 1* window is displayed.

  ![Export DATEV CVS: Step 1](../../Assets/Screenshots/RetailSuiteAccounting/Extras/Export/ExportDatevCVS01.png "[Export DATEV CVS: Step 1]")

> 2. Select the fiscal year to be exported in the *Fiscal year* drop-down list.

Reviewer: ```suggestion
2. Select the fiscal year to be exported in the *Fiscal year* drop-down list. All available fiscal years are displayed in the list.
```

> 3. Enter the months and journals to be exported in the *Period* field.

Reviewer: ```suggestion
3. Enter the period of months and journals to be exported in the *Months* and *Journals* fields.
```

> 4. Select the appropriate radio button (**Export all/All from last export on (if last export exists, otherwise all)**) in the *Selection* field. Mark the *Set marker to export time* checkbox, if necessary.

Reviewer: ```suggestion
4. Select the appropriate radio button in the *Selection* field. The following option are available:
  - **Export all**
  - **All from last export on (if last export exists, otherwise all)**    
 If desired, select the *Set marker to export time* checkbox.
```

> 5. Select the appropriate options in the *Impersonal accounts* and *Personal accounts* drop-down lists.
> 
>   > [Info] When selecting the *Custom* option, additional fields are displayed to customized your export.

Reviewer: ```suggestion
5. Select the appropriate options in the *Impersonal accounts* drop-down list. The following options are available:
  - **Only individually changed**:
  - **--- all ---**:
  - **Custom**:
  - **No**:

  > [Info] When selecting the **Custom** option, the *Custom* field to enter a custom account number or account number range and the *Export sub-postings* drop-down list to decide whether or not exporting the sub-postings are displayed.
  
6. Select the appropriate options in the *Personal accounts* drop-down list. The following options are available:
  - **Yes**:
  - **Custom**:
  - **No**:

  > [Info] When selecting the **Custom** option, the *Custom* field to enter a custom account number or account number range is displayed. 

```
Explain the options after the colon.

  ![Export DATEV CVS Custom](../../Assets/Screenshots/RetailSuiteAccounting/Extras/Export/ExportDatevCVS01cust.png "[Export DATEV CVS Custom]")

> 6. Select the appropriate option in the *Export only booking-relevant master data records* drop-down list.

Reviewer: ```suggestion
7. Click the *Export only posting-relevant master data records* drop-down list and select the appropriate option. The following options are available:
  - **Yes**:
  -**No**:
```
explain options

> 7. Select the appropriate option in the *Take Receipt 2 field in the export document from booking record field* drop-down list.

Reviewer: ```suggestion
7. Click the *Take Receipt 2 field in the export document from posting record field* drop-down list and select the appropriate option.
```
+ Welche Optionen werden angezeigt

>   > [Info] Here it can be determined which value will be displayed in the *Receipt 2* field in the export file.

Reviewer: ```suggestion
```

> 8. Select the appropriate option in the *Saved personal accounts length* drop-down list.

Reviewer: ```suggestion
9. Click the *Saved personal accounts length* drop-down list and select the appropriate option. The following options are available:
  - **Yes**:
  - **No**:
```

>   > [Info] This field displays automatically the personal account length preconfigured in the system. This is used to make sure that the personal account numbers are exported in full length.

Reviewer: ```suggestion
```
Info in Beschreibung de rOptionen einfließen lassen

>   > [Info] This field displays automatically the personal account length preconfigured in the system. This is used to make sure that the personal account numbers are exported in full length.

Reviewer: ```suggestion
10. If necessary, click the *Saved personal account length* drop-down list and select the appropriate option. By default, the personal account length preconfigured in the system is preselected. 

> [Info) It is important to select the appropriate length to make sure that the personal account numbers are exported in full length.
```
+ welche optionen werden angezeigt

> 9. Enter the applicable information in the *DATEV advisor number* and *DATEV client number*, if necessary.

Reviewer: ```suggestion
9. If necessary, enter the corresponding numbers in the *DATEV advisor number* and *DATEV client number* fields.
```

> 10. Click the [CONTINUE] button.  

Reviewer: ```suggestion
12. Click the [CONTINUE] button.  
```

A new window confirms the ongoing export process. When finished, a confirmation window is displayed.

  ![Export Datev CSV: Step 2](../../Assets/Screenshots/RetailSuiteAccounting/Extras/Export/ExportDatevCVS02.png "[Export Datev CSV: Step 2]")

> 11. Click the [OK] button.  
> The confirmation window closes. A new window shows the exported file(s) as a zip file.

Reviewer: ```suggestion
13. Click the [OK] button in the confirmation window.  
The confirmation window is closed. The exported file(s) are displayed as a zip file.
```

> 12. Click the zip file name.  
> The zip file is downloaded in your computer.   
> 
>   ![Export Datev CSV: Step 3](../../Assets/Screenshots/RetailSuiteAccounting/Extras/Export/ExportDatevCVS03.png "[Export Datev CSV: Step 3]")
> 
>   > [Info] Per default, the downloaded export file is saved in the Downloads folder of your computer.

Reviewer: ```suggestion
14. Click the zip file.  
The zip file has been downloaded.

  ![Export Datev CSV: Step 3](../../Assets/Screenshots/RetailSuiteAccounting/Extras/Export/ExportDatevCVS03.png "[Export Datev CSV: Step 3]")

  > [Info] By default, the downloaded export file is saved in the *Downloads* folder of your device.
```

## Export the data GDPdU-compliant

Accounting data can be exported into a GDPdU-compliant audit file format, a legal requirement for German companies on data access established by the German tax authorities.

#### Prerequisites

  - The accounting is configured via the configuration wizard, see [Run the accounting wizard](../Integration/01_RunAccountingWizard.md).
  - A fiscal year is selected, see [Select a fiscal year](./01_SelectFiscalYear.md).

#### Procedure

*Accounting > Select fiscal year > Extras > Tab EXPORT*

  ![Export](../../Assets/Screenshots/RetailSuiteAccounting/Extras/Export/Export.png "[Export]")

1. Click the [GDPdU EXPORT] button.    
The *Export: GDPdU Export: Step 1* window is displayed.

  ![Export GDPdU: Step 1](../../Assets/Screenshots/RetailSuiteAccounting/Extras/Export/ExportGDPdU01.png "[Export GDPdU: Step 1]")

> 2. Select the fiscal year to be exported in the *Fiscal year* drop-down list.

Reviewer: ```suggestion
2. Click the *Fiscal year* drop-down list and select the fiscal year to be exported. All available fiscal years are displayed in the list.
```

> 3. Enter the months and journals to be exported in the *Period* field.

Reviewer: ```suggestion
3. 3. Enter the period of months and journals to be exported in the *Months* and *Journals* fields.
```

> 4. Select the appropriate option in the *Personal accounts* drop-down list.

Reviewer: ```suggestion
4. Click the *Personal accounts* drop-down list and select the appropriate option. The following options are available:
  - **Yes**
  - **No**
```

> 5. Select the appropriate option in the *Export only booking-relevant master data records* drop-down list.

Reviewer: ```suggestion
5. Click the *Export only booking-relevant master data records* drop-down list and select the appropriate option. The following options are available:
  - **Yes**
  - **No**
```

6. Select the appropriate option in the *Saved personal accounts length* drop-down list.

  > [Info] This field displays automatically the personal account length preconfigured in the system. This is used to make sure that the personal account numbers are exported in full length.

7. Enter the applicable information in the *DATEV advisor number* and *DATEV client number*, if necessary.

8. Click the [CONTINUE] button.  
  A new window confirms the ongoing export process. When finished, a confirmation window is displayed.

  ![Export GDPdU: Step 2](../../Assets/Screenshots/RetailSuiteAccounting/Extras/Export/ExportGDPdU02.png "[Export GDPdU: Step 2]")

9. Click the [OK] button.  
  The confirmation window closes. A new window shows the exported file(s) as a zip file.

>   ![Export GDPdU: Step 3](../../Assets/Screenshots/RetailSuiteAccounting/Extras/Export/ExportGDPdU03.png "[Export GDPdU: Step 3]")

Reviewer: vgl. oben

## Create a PDF evaluation

Accounting data can be exported for business evaluation  purposes. The system offers a functionality to create and customize BWA reports. For detailed information, see [Manage reports](../Integration/07_ManageBWAReports.md).

#### Prerequisites

  - The accounting is configured via the configuration wizard, see [Run the accounting wizard](../Integration/01_RunAccountingWizard.md).
  - A fiscal year is selected, see [Select a fiscal year](./01_SelectFiscalYear.md).

#### Procedure

> *Accounting > Select fiscal year > Book > Context menu EVALUATIONS > Menu entry BWA*
> 
>   ![Evaluations](../../Assets/Screenshots/RetailSuiteAccounting/Book/EvaluationsBWA.png "[Evaluations]")
> 
> 1. Click the *BWA* menu entry.  
> The window *BWA* is displayed.  

Reviewer: ```suggestion
*Accounting > Select fiscal year > Post*

  ![Evaluations](../../Assets/Screenshots/RetailSuiteAccounting/Book/EvaluationsBWA.png "[Evaluations]")

1. Click the [EVALUATIONS] button in the header.
A context menu is displayed.

2. Click the *BWA* menu entry.  
The *BWA* window is displayed.  
```

  ![BWA](../../Assets/Screenshots/RetailSuiteAccounting/Book/BWA.png "[BWA]")

> 2. Select the appropriate option in the *Sorting* drop-down list.

Reviewer: ```suggestion
3. Click the *Sorting* drop-down list and select the appropriate option. The following options are available:
  - **normal**:
  - **by cost centre**:
```
+ Optionen erklären

> 3. If necessary, mark the *Print cost centre allocation* checkbox.

Reviewer: ```suggestion
3. If necessary, select the *Print cost centre allocation* checkbox. The checkbox is only unlocked, if the option **by cost centre** is selected in the *Sorting* drop-down list.
```

> 4. Select the months to be included in the *Accounting month* drop-down lists.

Reviewer: ```suggestion
4.  Click the *Accounting month from - to* drop-down lists and select the months to be included. All available months are displayed in the list.
```
Werden da die Monate oder die Journale angezeigt?

> 5. If applicable, select the years to be compared in the BWA report in the *Comparison year* and *Comparison year 2* drop-down lists.

Reviewer: ```suggestion
5. If applicable, select the years to be compared in the BWA report in the *Comparison year* and *Comparison year 2* drop-down lists.
```
```suggestion
5. If applicable, click the *Comparison year* and *Comparison year 2* drop-down lists and select the fiscal years to be compared in the BWA report. All available fiscal years are displayed in the list. If you want no fiscal year to be compared, select the **no year for comparison** option.
```

> 6. Select the BWA report in the *BWA number* drop-down list.

Reviewer: ```suggestion
6. Click the *BWA number* drop-down list and select the desired BWA report. All available BWA reports are displayed in the list.
```

>   > [Info] There are different preconfigured evaluation report types available in the system. They can be used as provided or completely customized. For detailed information, see [Manage reports](../Integration/07_ManageBWAReports.md).

Reviewer: ```suggestion
```
Benötige ich die Info an dieser Stelle?

> 7. Select one of the layout options in the *Layout* drop-down list.

Reviewer: ```suggestion
7. Click the *Layout* drop-down list and select the desired layout option. The following options are available:
  - **Month - cum. - cum. prior year**
  - **13 months cumulative**
  - **Prior-year comparison**
  - **13 month with comparison**
```

> 8. Select the appropriate option in the *Show account* field.

Reviewer: ```suggestion
8. Click the *Show account* drop-down list and select the appropriate option. The following options are available:
  - **No**
  - **Yes**
```

9. Click the [PRINT] button.  

> A new window confirms the ongoing PDF creation process. When finished, the PDF file opens automatically.

Reviewer: ```suggestion
The PDF creation has been started. The *Printing BWA* window is displayed. An *Printing BWA* notice and a loading icon are displayed in the window.  When finished, the PDF file is displayed in the window.
```
